pkgs/test/doc/configuration.md

```diff
@@ -24,46 +24,48 @@ tags:
   sanity:
 ```
 
-* [Test Configuration](#test-configuration)
-  * [`timeout`](#timeout)
-  * [`verbose_trace`](#verbose_trace)
-  * [`chain_stack_traces`](#chain_stack_traces)
-  * [`js_trace`](#js_trace)
-  * [`skip`](#skip)
-  * [`retry`](#retry)
-  * [`test_on`](#test_on)
-* [Runner Configuration](#runner-configuration)
-  * [`include`](#include)
-  * [`paths`](#paths)
-  * [`filename`](#filename)
-  * [`names`](#names)
-  * [`plain_names`](#plain_names)
-  * [`include_tags`](#include_tags)
-  * [`exclude_tags`](#exclude_tags)
-  * [`platforms`](#platforms)
-  * [`concurrency`](#concurrency)
-  * [`pause_after_load`](#pause_after_load)
-  * [`run_skipped`](#run_skipped)
-  * [`pub_serve`](#pub_serve)
-  * [`reporter`](#reporter)
-  * [`fold_stack_frames`](#fold_stack_frames)
-  * [`custom_html_template_path`](#custom_html_template_path)
-* [Configuring Tags](#configuring-tags)
-  * [`tags`](#tags)
-  * [`add_tags`](#add_tags)
-* [Configuring Platforms](#configuring-platforms)
-  * [`on_os`](#on_os)
-  * [`on_platform`](#on_platform)
-  * [`override_platforms`](#override_platforms)
-  * [`define_platforms`](#define_platforms)
-  * [Browser/Node.js Settings](#browser-and-node-js-settings)
-    * [`arguments`](#arguments)
-    * [`executable`](#executable)
-    * [`headless`](#headless)
-* [Configuration Presets](#configuration-presets)
-  * [`presets`](#presets)
-  * [`add_preset`](#add_preset)
-* [Global Configuration](#global-configuration)
+- [Test Configuration](#test-configuration)
```

Comment:
Apparently my markdown plugin did this 🤣 . Should I revert? I am tempted to accept the change so I don't have to revert it every time I touch this file... lol.

Reply:
I'd be fine with a 1 time reformat to this style if it will make it easier in future PRs.

I don't want to submit this in the same commit though. Can you open a separate PR?

Reply:
I was able to disable this auto-formatting in my plugin 👍